js/src/features/project/epics.ts

```diff
@@ -23,7 +24,8 @@ import {
   addProjectTeamMemberAsync,
   closeProjectTeamNewMemberFormAction,
 } from "./actions";
-import { push } from "connected-react-router";
+import { notificationError } from "../notification/actions";
+import { SOMETHING_WENT_WRONG } from "../../constants";
```

Comment:
Better to put in notifications like default error notification (if you don't pass any argument)

```ts
dispatch(notificate({ type: 'error' }) // the same as `dispatch(notificate({ type: 'error', message: SOMETHING_WENT_WRONG })` 
```

Reply:
Done